CMSgov/mdct-mcr
# add error handling

## Description

- Add `ErrorAlert` component and related tests.
- Delete `errorHandler` utility function and instead handle errors locally.
- Wire up alert component in needed locations.
- Fix up _a bunch_ of tests.

Partially satisfies [MDCT-259: Error boundary & Alert component](https://qmacbis.atlassian.net/browse/MDCT-259)

Note: Will have a second PR with the error boundary stuff, but this one was getting big and I wanted to keep it focused on a single topic.

### How to test

1. Pull and run locally
2. Easiest way to see an example is to attempt cognito login with incorrect credentials.
3. Look through all the fun comments I left you in the code, you lucky reviewer! 🎉 

### Changed Dependencies

No changed deps.

## Code author checklist

## Reviewer checklist (two different people)
